python/graphstorm/model/edge_decoder.py

```diff
@@ -224,50 +279,44 @@ def predict_proba(self, g, h, e_h=None):
 
     @property
     def in_dims(self):
-        """ The number of input dimensions.
-
-        Returns
-        -------
-        int : the number of input dimensions.
+        """ Return the input dimension size, which is given in class initialization.
         """
         return self.in_units
 
     @property
     def out_dims(self):
-        """ The number of output dimensions.
-
-        Returns
-        -------
-        int : the number of output dimensions.
+        """ Return the output dimension size. If this decoder is set for edge regression,
+        will return ``1``.
         """
         return 1 if self.regression else self.num_classes
 
 
 class MLPEdgeDecoder(GSEdgeDecoder):
-    """ MLP based edge classificaiton/regression decoder
+    """ MLP-based decoder for edge prediction tasks.
 
     Parameters
     ----------
-    h_dim : int
-        The input dim of decoder. It is the dim of source or destinatioin node embeddings.
-    out_dim : int
-        Output dim. e.g., number of classes
-    multilabel : bool
-        Whether this is a multilabel classification.
-    target_etype : tuple of str
-        Target etype for prediction
+    h_dim: int
+        The input dimension size. It is the dimension for both source and destinatioin
```

Comment:
```suggestion
        The input dimension size. It is the dimension for both source and destination
```

Reply:
There may be multiple typos here about ``destinatioin``. I just point it out all I found here.